composer.json

```diff
@@ -4,24 +4,25 @@
     "type": "symfony-bundle",
     "license": "MIT",
     "require": {
-        "php": ">=8.1",
+        "php": ">=8.1|^8.2",
         "ext-json": "*",
-        "guzzlehttp/guzzle": "^6.0 || ^7.0",
-        "psr/log": "^1.1",
-        "symfony/http-kernel": "^6.1",
-        "symfony/property-info": "^6.1",
-        "symfony/serializer": "^6.1",
-        "webmozart/assert": "^1.3"
+        "guzzlehttp/guzzle": "^6.5|^7.9.2",
+        "psr/log": "^1.1.4",
+        "symfony/http-kernel": "6.4.*",
+        "symfony/property-access": "6.4.*",
+        "symfony/property-info": "6.4.*",
+        "symfony/serializer": "6.4.*",
+        "webmozart/assert": "^1.11"
```

Comment:
if this is not require for this PR don't bump min versions in `require`section
This is public library, maybe someone has symfony 6.1 for example